src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/config/RedisConfiguration.kt

```diff
@@ -31,6 +34,9 @@ class RedisConfiguration {
     return RedisCacheManagerBuilderCustomizer { builder: RedisCacheManagerBuilder ->
       builder.clientCacheFor<List<StaffMember>>("staffMembersCache", Duration.ofHours(6), version, objectMapper)
         .clientCacheFor<StaffMember>("staffMemberCache", Duration.ofHours(6), version, objectMapper)
+        .clientCacheFor<OffenderDetailSummary>("offenderDetailsCache", Duration.ofMinutes(20), version, objectMapper)
+        .clientCacheFor<UserOffenderAccess>("userAccessCache", Duration.ofMinutes(20), version, objectMapper)
+        .clientCacheFor<InmateDetail>("inmateDetailsCache", Duration.ofMinutes(20), version, objectMapper)
```

Comment:
should these expiry values be defined in a config file perhaps?

Reply:
Have extracted into properties